server/src/ormconfig.ts

```diff
@@ -10,7 +9,7 @@ const config: ConnectionOptions = {
   password: process.env.RSSHCOOL_API_PG_PASSWORD,
   database: process.env.RSSHCOOL_API_PG_DATABASE,
   entities: models,
-  migrations,
+  migrations: ['src/migrations/*.ts'],
```

Comment:
@apalchys is there any reason for having the `index` file? I removed it: nasty to update it all the times once you add migrations

Reply:
if i remember correctly, it doesn't work in production mode (container) with current setup because we don't have typescript and ts files inside container. but need to check

Reply:
understood. I have reverted for now. will check later and if ok, commit as separate change.